src/migrations/20231030091931-add-created-by-and-status-change-request-schedule.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+exports.up = function (db, cb) {
+  db.runSql(`
+        ALTER TABLE change_request_schedule
+            ADD COLUMN IF NOT EXISTS created_by integer REFERENCES users(id);
+
+        ALTER TABLE change_request_schedule
+            ADD COLUMN IF NOT EXISTS status text default 'pending';
```

Comment:
Would we wanna use enum types for this or is text preferable? How do we do it in other places?

Reply:
makes sense

Reply:
Sure. I think the most important bit is to stay consistent with how we do it other places, so I'd check for that first, and do what we do there. If we have no precedents for this, I think an enum makes sense. But don't trust me blindly on that 😅

Reply:
https://github.com/Unleash/unleash/commit/6875daf15768d026a53bbc1916fb5c4219fe5680#commitcomment-131584329